nextcord/client.py

```diff
@@ -2102,33 +2117,170 @@ def get_application_command(self, command_id: int) -> Optional[BaseApplicationCo
         """
         return self._connection.get_application_command(command_id)
 
-    def get_application_command_from_signature(
-        self, name: str, cmd_type: Union[int, ApplicationCommandType], guild_id: Optional[int]
-    ) -> Optional[BaseApplicationCommand]:
+    def __actual_get_application_command_from_signature(
+        self,
+        *,
+        type: Union[int, ApplicationCommandType],
```

Comment:
I'm a little iffy about `type` being used, as that shadows a built-in Python function.

Reply:
I don't see a problem with using `type` here. The builtin `type` isn't used in this function and I don't see why it would be used since all this does is call `ConnectionState.get_application_command_from_signature`.